pkg/auth0-cli-config-generator/main.go

```diff
@@ -12,8 +12,9 @@ import (
 	"path"
 	"path/filepath"
 	"strings"
-	"time"
 
+	"github.com/auth0/auth0-cli/internal/auth"
```

Comment:
can we export the configuration bits more formally? If the config-generator wouldn't need to import /internal/. 
what's the minimum structs or functions that are needed from the CLI config to be initialized externally?

Reply:
what if we actually provide a cli command such as `$auth config init` for what we're trying to accomplish from the config-generator? that command would skip all the shared config init (like `help`). The config generator ignores all the details. And we provide CLI users a way to do the same if they want to use the CLI from an automated scenario

Reply:
Took a shot at what I think you are suggesting, let me know what you think.

Reply:
this is great. It's looking pretty good. I left some random comments. I can fully review it later, but thanks for doing this 🙇🏼 
integration tests are passing already using the command? 😮